web-common/src/features/connectors/code-utils.ts

```diff
@@ -20,6 +25,18 @@ type: connector
 
 driver: ${connector.name}`;
 
+  // Use the provided orderedProperties if available, otherwise fall back to configProperties/sourceProperties
+  let properties =
+    options?.orderedProperties ??
+    connector.configProperties ??
+    connector.sourceProperties ??
+    [];
+
+  // Optionally filter properties
+  if (options?.fieldFilter) {
+    properties = properties.filter(options.fieldFilter);
+  }
```

Comment:
This complexity still makes me nervous, but let's just take it as motivation to move to the JSON Schema going forward